go/vt/vtadmin/cluster/discovery/discovery_static_file.go

```diff
@@ -175,3 +200,68 @@ func (d *StaticFileDiscovery) DiscoverVTGates(ctx context.Context, tags []string
 
 	return results, nil
 }
+
+// DiscoverVtctld is part of the Discovery interface.
+func (d *StaticFileDiscovery) DiscoverVtctld(ctx context.Context, tags []string) (*vtadminpb.Vtctld, error) {
+	vtctlds, err := d.DiscoverVtctlds(ctx, tags)
+	if err != nil {
+		return nil, err
+	}
+
+	count := len(vtctlds)
+	if count == 0 {
+		return nil, ErrNoVTGates
```

Comment:
Sorry for missing this before -- I think this should be `ErrNoVtctlds`.

Reply:
Fixed! Nice catch!